src/preview/index.js

```diff
@@ -27,17 +27,20 @@ const withData = (ThemeProvider, { providerFn, onThemeSwitch }) => {
     theme: store => store.themesList[store.currentTheme || 0],
     themeInd: store => store.currentTheme,
     onThemeSwitch: () => onThemeSwitch,
+
   })(DecoratorUI(CurrentThemeProvider), { isGlobal: true });
 };
 
 export const withThemes = (
   ThemeProvider,
   themesList,
   { providerFn, onThemeSwitch = onThemeSwitchDefault } = {},
+  snippetFunc,
```

Comment:
This is good. But I'd like to align this with a future planning features and with the current API style. So let's update it in the following way:

1. Move it into the third agrument, so it's more convinient to keep optional configs in a single place
2. Let's sligtly update a name. We plan to have the similar feature for the values panel, so I'd like it to be a bit more specific e.g. `fieldSnippetFn`
3. We can pass a default value right here and rely on this in other places

Reply:
Done

Reply:
it should be

```
{
      providerFn,
      onThemeSwitch = onThemeSwitchDefault,
     fieldSnippetFn  = fieldSnippetFnDefault,
}
```